code/game/objects/items/trash.dm

```diff
@@ -153,7 +153,12 @@
 	desc = "An empty wrapper from a barcardine bar. You notice the inside has several medical labels. You're not sure if you care or not about that."
 	icon_state = "barcardine_trash"
 
-/obj/item/trash/mre
+/obj/item/trash/berrybar
+	name = "berry bar wrapper"
+	desc = "An empty wrapper from a berry bar. You notice it has writings about how tritium sucks and stinks but You're not sure if you care or not about that."
```

Comment:
I think you forgot to change this description

Reply:
oopsie